pkg/config/config.go

```diff
@@ -66,53 +66,13 @@ const (
 var (
 	// DefaultImage is the URL of the docker image to run for the aggregator and workers
 	DefaultImage = "sonobuoy/sonobuoy:" + buildinfo.Version
+
 	// DefaultResources is the default set of resources which are queried for after plugins run. The strings
 	// are compared against the resource.Name given by the client-go discovery client. The non-standard values
 	// that are included here are: podlogs, servergroups, serverversion. The value 'nodes', although a crawlable
 	// API value, also is used to query against the healthz and configz endpoints on the node.
-	DefaultResources = []string{
```

Comment:
Are you dynamically discovering the resources, is that why these are no longer needed?

Reply:
Yes we dynamically walk the API server. That code isn't easy to follow but thats been done for a long time. The difference is that prior to this change we dynamically discover all the ytpes and then only get this explicit list of types. Now we'll default to getting them all (except for secrets).